code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -367,12 +370,6 @@
 	name = "Maintenance Drugs"
 	chemical_flags = NONE
 
-/datum/reagent/drug/pumpup/on_mob_metabolize(mob/living/carbon/L)
-	var/obj/item/organ/internal/liver/liver = L.get_organ_slot(ORGAN_SLOT_LIVER)
-	if(HAS_TRAIT(liver, TRAIT_MAINTENANCE_METABOLISM))
-		L.add_mood_event("maintenance_fun", /datum/mood_event/maintenance_high)
-		metabolization_rate *= 0.8
-
```

Comment:
i think this is meant to be for the maintenance drugs, they just forgot to change the typepath

Reply:
ye

Reply:
Yeah that checks out.